icu4j/main/translit/src/main/java/com/ibm/icu/text/LowercaseTransliterator.java

```diff
@@ -54,16 +52,14 @@ public LowercaseTransliterator(ULocale loc) {
         super(_ID, null);
         locale = loc;
         csp=UCaseProps.INSTANCE;
-        iter=new ReplaceableContextIterator();
-        result = new StringBuilder();
         caseLocale = UCaseProps.getCaseLocale(locale);
     }
 
     /**
      * Implements {@link Transliterator#handleTransliterate}.
      */
     @Override
-    protected synchronized void handleTransliterate(Replaceable text,
+    protected void handleTransliterate(Replaceable text,
```

Comment:
Please do the same thing for UppercaseTransliterator, TitlecaseTransliterator, CaseFoldTransliterator, BreakTransliterator, all in the same folder.